apps/src/templates/Certificate.jsx

```diff
@@ -109,7 +106,7 @@ export default class Certificate extends Component {
           {i18n.congratsCertificateHeading()}
         </h1>
         <LargeChevronLink
```

Comment:
Wait. We're still showing this link no matter what? I though the plan was to hide if we don't navigate to congrats from a tutorial.

Reply:
Sorry, you're 100% right.  Thank you for fixing this!  Related PR: https://github.com/code-dot-org/code-dot-org/pull/19263